mule-bpm-module/src/main/java/com/alfame/esb/bpm/module/internal/impl/BPMSchemaOperationsProcessEngineBuild.java

```diff
@@ -0,0 +1,100 @@
+package com.alfame.esb.bpm.module.internal.impl;
+
+import org.apache.ibatis.session.SqlSession;
+import org.flowable.common.engine.api.FlowableWrongDbException;
+import org.flowable.common.engine.impl.db.DbSqlSession;
+import org.flowable.common.engine.impl.db.SchemaManager;
+import org.flowable.common.engine.impl.interceptor.CommandContext;
+import org.flowable.engine.ProcessEngineConfiguration;
+import org.flowable.engine.impl.SchemaOperationsProcessEngineBuild;
+import org.flowable.engine.impl.util.CommandContextUtil;
+import org.flywaydb.core.Flyway;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class BPMSchemaOperationsProcessEngineBuild extends SchemaOperationsProcessEngineBuild {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BPMSchemaOperationsProcessEngineBuild.class);
+
+    @Override
+    public Void execute(CommandContext commandContext) {
+        Void returnValue = null;
+
+        String databaseSchemaUpdate = CommandContextUtil.getProcessEngineConfiguration().getDatabaseSchemaUpdate();
+
+        if (!ProcessEngineConfiguration.DB_SCHEMA_UPDATE_FALSE.equals(databaseSchemaUpdate)) {
+            SchemaManager schemaManager = CommandContextUtil.getProcessEngineConfiguration(commandContext).getSchemaManager();
+            DataSource dataSource = CommandContextUtil.getProcessEngineConfiguration().getDataSource();
+
+            if (dataSource != null) {
+                Connection connection = null;
+                try {
+                    connection = dataSource.getConnection();
```

Comment:
Maybe use try-with-resources for connection?

Reply:
Leaving current (verified solution) as there is no functional issues with the old style implementation.